### PR TITLE
Move q-bar to top (header redesign)

### PR DESCRIPTION
**Pull request by manuelroth**
- This PR moves the q-bar to the top and integrates the status-bar into q-bar
- The q-bar is now part of each page. It is customized for each page using the custom element slot feature. For example the index page doesn't have the back button and feed/item-detail/editor pages don't have the logo but show the back button
- This branch is deployed on staging and q-beta for testing -> https://q.st-staging.nzz.ch/index

### CODE REVIEW

**Comment by benib** [comment]
Many thanks. I will merge this once we are sure we want to release that layout change to keep `dev` clean for other things in the meantime.

File: client/src/styles/config.css [hunk 1]
```diff
@@ -62,5 +62,5 @@
   --q-link-color: var(--q-color-primary-5);
 
   /* layout */
-  --q-space-base: 16px;
+  --q-space-base: 8px;
```
Comment (benib) [inline client/src/styles/config.css]: This should be either kept at 16px or the calculation needs to be adjusted everywhere where this is used.

File: client/src/elements/organisms/tool-status-bar.html [hunk 2]
```diff
@@ -7,9 +7,8 @@
   </div>
 
   <div class="tool-status-bar__group">
-    <icon-active-state is-active.bind="item.conf.active" class="tool-status-bar__item" ></icon-active-state>
+    <icon-active-state is-active.bind="item.conf.active" class="tool-status-bar__item"></icon-active-state>
     <button-secondary if.bind="item.conf.active" class="tool-status-bar__item" icon="deactivate" click.delegate="onDeactivateClick()"></button-secondary>
```
Comment (benib) [inline client/src/elements/organisms/tool-status-bar.html]: as there is enough space, we should add the activate/deactivate text to this button.
same for the save button.

File: client/src/styles/config.css (hunk 1 above)
Comment (manuelroth) [reply client/src/styles/config.css]: As we discussed I moved back space-base back to 16px -> https://github.com/nzzdev/Q-editor/pull/116/commits/33610ad844819d195f60a4841c18d6049b313742

File: client/src/elements/organisms/tool-status-bar.html (hunk 2 above)
Comment (manuelroth) [reply client/src/elements/organisms/tool-status-bar.html]: Text is now shown next to the icon if there is enough space available (won't be show on small screen sizes) -> https://github.com/nzzdev/Q-editor/pull/116/commits/81d6d13d2feffb337495807de9be6deb35d1c331

**Comment by manuelroth** [comment]
I have redeployed this branch on staging with the latest. A separate PR with the editor options sidebar based on this branch will follow

**Pull request merged**